cypress/page/products-list.page.ts

```diff
@@ -3,8 +3,11 @@ class ProductslistPage {
     private proceedToCheckoutProduct: string;     
 
     constructor() {
-        this.productList ="#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-        this.proceedToCheckoutProduct ="[style*='display: block;'] .button-container > a";
+        this.productList ="#center_column> ul > li > div a[title='Add to cart']";
+	    // Here, this selector was improved in order to avoid using classes.
+        this.proceedToCheckoutProduct ="[style*='display: block;'] [title='Proceed to checkout']";
+	    // Here, this selector was improved in order to avoid using classes.
+        // Besides, this is more clear for the one who is reading it.
```

Comment:
probably `a[title='Proceed to checkout']` or ` .button-container a[title='Proceed to checkout']` should work

Reply:
Ok.

Here I have a question, isn´t it better to have two attributes than a tag and an attribute or a class, tag and attribute?
But, maybe in this case the problem is that the first attribute seems to be very general, right? I am a little bit confused with this one, because I don´t even know how to get **[style*='display: block;']** so that´s why I just left it.

Reply:
in this case the first is not readable, and also is not a good practice, since the attribute value is not something that can led us to understand which element we want to interact, also seems that the attribute value can have probabilities to be changed in the future since it seems a css function